Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -187,8 +188,13 @@ class WebTreeView
           let hasChildren = activeTreeNode.getAttribute ('aria-expanded') === 'true';
           if (hasChildren)
           {
-            const newTreeNode = activeTreeNode.querySelector<HTMLElement> (':scope > ul > li')!;
-            WebTreeView.SetFocus (newTreeNode, activeTreeNode);
+            const nodeIndex = treeNodes.indexOf(activeTreeNode);
+            const liNode = parentTreeNodeListItems[nodeIndex];
+            const newListItemChildNode = liNode.querySelector<HTMLElement> (':scope > ul > li')!;
```

Comment:
In theory, this CSS selector could return any of the `li` elements if there are many. To make it more apparent that we want the first I would add the `:first-child` selector as well.